pkg/container/container.go

```diff
@@ -429,3 +435,47 @@ func (c *container) SelinuxLabel(sboxLabel string) ([]string, error) {
 	}
 	return ret, nil
 }
+
+// AddUnifiedResources adds the cgroup-v2 resources specified in the io.kubernetes.cri-o.UnifiedCgroup annotation
+func (c *container) AddUnifiedResources(specgen *generate.Generator, sb *sandbox.Sandbox, containerConfig *types.ContainerConfig) error {
```

Comment:
nit: the spec generator and container config are already members, we don't need to pass them in :)

Reply:
I also think we could just pass in the annotations, which will make unit testing easier (don't have to initialize a whole sandbox). If you make this change, I'd also change the function to `AddUnifiedResourcesFromAnnotation` or something

Reply:
thanks!  Amended in the last version